process/mockoon-usage.md

```diff
@@ -0,0 +1,317 @@
+# Mockoon Usage
```

Comment:
@LakshayaT  Use this link to generate better quality pipeline images. 
https://lucid.app/lucidchart/a653ad15-e662-4102-8739-036b6d77bd6a/edit?viewport_loc=-1101%2C16%2C1930%2C1032%2C0_0&invitationId=inv_1ede69a0-72b4-474a-b991-eaf9c1c60744

Reply:
Addressed in https://github.com/OsmosysSoftware/dev-standards/pull/49/commits/2beeb35f34d00bdc66e5a43ec2d625671ae8227a